src/ast.js

```diff
@@ -16,7 +16,9 @@ export class Expression {
   }
 
   toString() {
-    return Unparser.unparse(this);
+    return typeof FEATURE_NO_UNPARSER === 'undefined' ? 
```

Comment:
This is not the problem. Reading a non-existent variable is an error in strict mode.
`FEATURE_NO_UNPARSER === 'undefined'` will throw if `FEATURE_NO_UNPARSER` does not exist on `window`, which is the common case.

The direct comparison to `undefined` is only safe on existing objects, existing vars or parameters.